theme/tuxlite_tbs/templates/base.html

```diff
@@ -144,7 +144,7 @@
                 </li>
            
                 {% for name, link in SOCIAL %}
-                <li><a href="{{ link }}">{{ name }}</a></li>
```

Comment:
SOCIAL looks unused in the pelicanconf, fwiw

Reply:
I tried that, but a) it was in a section under the Pyninsula logo and I wasn't sure if consolidating them together was an intentional choice, and b) the "socials" section wants to put image logos next to things and includes an *ancient* twitter logo, so fixing that and adding a mastodon logo was a bit more than a 5 minute PR 😅

That said, I'm happy to move them to the socials category instead if that's preferred.

Reply:
No need to move anything, just observing.